R/auth.R

```diff
@@ -0,0 +1,237 @@
+#' Authentication options
+#' 
+#' @description 
+#' There are three ways that you can authenticate with the twitter API:
+#' 
+#' * `auth_user()` interactively authenticates an existing twitter user. 
+#'   This form is most appropriate if you want rtweet to control your
+#'   twitter account.
+#'   
+#' * `auth_app()` authenticates as a twitter application. An application can't 
+#'    perform actions (i.e. it can't tweet) but otherwise has generally higher 
+#'    rate limits (i.e. you can do more searches) generally higher. This form is 
```

Comment:
```suggestion
#'    rate limits (i.e. you can do more searches). This form is 
```

Reply:
Thanks; added manually since I've made a bunch of other changes.